packages/http-client/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@typespec/http-client",
+  "version": "0.1.0",
+  "type": "module",
+  "main": "dist/src/index.js",
+  "tspMain": "lib/main.tsp",
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
```

Comment:
I think for new libraries specially should chang ethis to be `import` as teh correct key as otherwise with default it could get resolved as commonjs and I think you can also drop `types`(check tspd package)